cypress-tests/cypress/e2e/PaymentUtils/Adyen.js

```diff
@@ -398,6 +398,24 @@ export const connectorDetails = {
         },
       },
     },
+    MITAutoCapture: {
+      Request: {},
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+        },
+      },
+    },
+    MITManualCapture: {
+      Request: {},
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_capture",
```

Comment:
do we plan to add more validations in here?

Reply:
We can add the validations and we should map the error messages also

Reply:
when? subsequent prs?

Reply:
It's added for another connector , here for this connector it's not needed